sljit_src/sljitNativeLOONGARCH_64.c

```diff
@@ -3625,7 +3627,10 @@ SLJIT_API_FUNC_ATTRIBUTE sljit_s32 sljit_emit_atomic_load(struct sljit_compiler
 	CHECK_ERROR();
 	CHECK(check_sljit_emit_atomic_load(compiler, op, dst_reg, mem_reg));
 
-	if (!(LOONGARCH_CFG2_LAMCAS & get_cpu_features(GET_CFG2))) {
+	if (op & SLJIT_ATOMIC_USE_CAS && !LOONGARCH_SUPPORT_AMCAS)
```

Comment:
I prefer `()` around & to make it clearer. Similar cases below.

Reply:
Added.